ceph/datadog_checks/ceph/ceph.py

```diff
@@ -70,7 +70,9 @@ def _collect_raw(self, ceph_cmd, ceph_cluster, instance):
             name = cmd.replace(' ', '_')
             raw[name] = res
 
-        mon_map = raw['status']['monmap']
+        mon_map = raw.get('status', {}).get('monmap')
+        if mon_map is None:
+            raise RuntimeError("Could not detect if ceph is an octopus release or not")
```

Comment:
```suggestion
            raise RuntimeError("Could not detect Ceph release series")
```
Ceph docs refers to Luminous/Nautilus/Octopus as series
https://docs.ceph.com/en/latest/releases/octopus/